uk.co.compucorp.civicrm.tasksassignments/js/src/crm-tasks-workflows/controllers/casetype.extended.controller.js

```diff
@@ -26,7 +37,7 @@
      */
     function addActivity (activitySet, activityType) {
       // call parent
```

Comment:
Could you remove this comment, it is not needed anymore.

Reply:
thanks! crept from another PR that can't be merged for now